src-tauri/src/node_adapter.rs

```diff
@@ -109,8 +109,8 @@ impl ProcessAdapter for MinotariNodeAdapter {
             if cfg!(target_os = "windows") {
                 // No need
             } else {
-                args.push("-p".to_string());
-                args.push("use_libtor=false".to_string());
+                // args.push("-p".to_string());
+                // args.push("use_libtor=false".to_string());
```

Comment:
Do we use libtor on linux too? I see inconsistency with main.rs

Reply:
I fixed this issue on my separate branch. We use libtor only for macos with missing ui changes:
https://github.com/tari-project/universe/pull/991